openedx/features/course_search/static/course_search/js/views/search_results_view.js

```diff
@@ -75,31 +82,33 @@
 
             showLoadingMessage: function() {
                 this.doCleanup();
-                this.$el.html(this.loadingTemplate());
+                HtmlUtils.setHtml(this.$el, HtmlUtils.template(this.loadingTemplate)());
                 this.showResults();
             },
 
             showErrorMessage: function() {
-                this.$el.html(this.errorTemplate());
+                HtmlUtils.setHtml(this.$el, HtmlUtils.template(this.errorTemplate)());
                 this.showResults();
             },
 
             doCleanup: function() {
-            // Empty any loading/error message and empty the el
-            // Bookmarks share the same container element, So we are doing
-            // this to ensure that elements are in clean/initial state
+                // Empty any loading/error message and empty the el
+                // Bookmarks share the same container element, So we are doing
```

Comment:
I don't fully understand this comment.  Is it still accurate regarding 'Bookmarks'?